inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -376,6 +376,49 @@ void FrontEnd::parseLayer(const Model& model, const ie::CNNLayerPtr& layer, cons
     }
 }
 
+void FrontEnd::processTrivialCases(const Model& model) {
+    std::map<ie::DataPtr, DataVector> ieToVpuDataVector;
```

Comment:
Nitpick: name is not descriptive. Try to explain meaning of information you store, do not repeat what already said in type

Reply:
Changed to `vpuDatasSortedByOrigData`.

Reply:
Don't use `datas`, use `dataObjects`

Reply:
Got it, fixed.